modulus/distributed/__init__.py

```diff
@@ -27,3 +27,8 @@
     reduce_loss,
     unmark_module_as_shared,
 )
+
+from .shard_tensor import ShardTensor, scatter_tensor
+
+# Load and register custom ops:
+from .custom_ops import *
```

Comment:
Should probably try and avoid the wildcard imports, if it's not too much trouble

Reply:
Good idea, resolved.